model/http/spans.yaml

```diff
@@ -52,7 +69,14 @@ groups:
     type: span
     extends: attributes.http.server
     span_kind: server
-    brief: 'Semantic Convention for HTTP Server'
+    brief: >
+      This span represents an inbound HTTP request.
+    note: |
+      **Span name:** refer to the [Span Name](/docs/http/http-spans.md#name) section.
```

Comment:
Will the link checker fail when/if these anchor links somehow get "broken"? If not, I'm not sure it is a good idea to start adding these to the yaml, as they are also not auto-generated.

Reply:
yep, our link checker (lychee) checks it now and we have a bunch of links from yaml to docs anyway.